Dockerfile.indexserver

```
@@ -1,7 +1,7 @@
 FROM alpine
 
 RUN apk update --no-cache && apk upgrade --no-cache && \
-    apk add --no-cache ca-certificates bind-tools tini git python3
+    apk add --no-cache ca-certificates bind-tools tini git python3 expat~=2.4.3
```

Suggesting a fix, but also wanted to mention: in https://github.com/sourcegraph/sourcegraph/pull/29493 we started changing dependencies to be `>=` instead, to avoid spurious build failures. I'm not sure that's ideal, but maybe a `expat=~2.4` would be better here to allow patch upgrades?
```suggestion
    apk add --no-cache ca-certificates bind-tools tini git python3 expat=~2.4.3
```